serverctl/tls.go

```diff
@@ -0,0 +1,105 @@
+package serverctl
```

Comment:
why in package serverctl rather than adding to package tls

Reply:
The new functions Use the database package, and TLS package gets imported by netclient. This avoids adding the database package as a dependency to netclient